kyuubi-common/src/test/scala/org/apache/kyuubi/service/authentication/JdbcAuthenticationProviderImplSuite.scala

```diff
@@ -0,0 +1,143 @@
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kyuubi.service.authentication
+
+import java.nio.file.Path
+import java.sql.{Connection, DriverManager}
+import java.util.Properties
+import javax.security.sasl.AuthenticationException
+
+import org.apache.kyuubi.{KyuubiFunSuite, Utils}
+import org.apache.kyuubi.config.KyuubiConf
+import org.apache.kyuubi.config.KyuubiConf._
+
+class JdbcAuthenticationProviderImplSuite extends KyuubiFunSuite {
+  protected val dbUser: String = "bowenliang123"
+  protected val dbPasswd: String = "bowenliang123"
+  protected var jdbcUrl: String = _
+
+  protected val authUser: String = "liangtiancheng"
+  protected val authPasswd: String = "liangtiancheng"
+
+  protected var conf = new KyuubiConf()
+  var conn: Connection = _
+  var authDb: Path = _
+
+  override def beforeAll(): Unit = {
+    super.beforeAll()
+
+    val datasourceProperties = new Properties()
+    datasourceProperties.put("user", dbUser)
+    datasourceProperties.put("password", dbPasswd)
+
+    authDb = Utils.createTempDir(namePrefix = getClass.getSimpleName)
+    authDb.toFile.delete()
+
+    jdbcUrl = s"jdbc:derby:;databaseName=$authDb;create=true"
+    conn = DriverManager.getConnection(
+      jdbcUrl
+        + ";user=" + dbUser
+        + ";password=" + dbPasswd,
+      datasourceProperties)
+
+    conn.prepareStatement("CREATE SCHEMA " + dbUser).execute();
+
+    conn.prepareStatement("CREATE TABLE user_auth (" +
+      "username VARCHAR(64) NOT NULL PRIMARY KEY, " +
+      "passwd VARCHAR(64))").execute();
+
+    val insertStmt = conn.prepareStatement("INSERT INTO user_auth " +
+      "(username, passwd) VALUES (?,?)")
+    insertStmt.setString(1, authUser)
+    insertStmt.setString(2, authPasswd)
+    insertStmt.execute();
+
+    conf = genJdbcAuthConfigs
+  }
+
+  override def afterAll(): Unit = {
+    super.afterAll()
+
+    // shutdown derby database
+    try {
+      DriverManager.getConnection(s"jdbc:derby:;databaseName=$authDb;shutdown=true")
+    } catch {
+      case e: Throwable =>
+    }
+  }
+
+  test("authenticate tests") {
+    var providerImpl = new JdbcAuthenticationProviderImpl(conf)
+
+    providerImpl.authenticate(authUser, authPasswd)
+
+    val e1 = intercept[AuthenticationException](providerImpl.authenticate("", ""))
```

Comment:
```suggestion
    val e1 = intercept[AuthenticationException] {
      providerImpl.authenticate("", "")
    }
```

Reply:
ok. changed to your suggested style. the previous style refered to `LdapAuthenticationProviderImplSuite`.